src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.3.0
+-----
+
+ * added `%env(catch:...)%` processor to catch exception and fallback to a default value
```

Comment:
it's `(catch:...:DEFAULT_KEY)` right

Reply:
to be consistent with `added `%env(key:...)%` processor`